### DISCUSSION TITLE
Pixels contain different values than what is exported?

### DISCUSSION THREAD

**Discussion by Raildex** [dlemstra/Magick.NET#1439] [Help] [answered]
I am using ImageMagick to read and write exr files and transfer data to an external encoder.
I noticed that the exr files Imagemagick writes contain the correct values.

When I try to read the pixels myself, they are extraordinarily large.

While the exr file has values between 0.0 and 1.0, the pixel value in the program is withint the 1000.0 and higher.
![a.exr inspector](https://github.com/dlemstra/Magick.NET/assets/14938934/a538e192-92fd-4183-9f00-7a286079cb28)
![grafik](https://github.com/dlemstra/Magick.NET/assets/14938934/e1cfef7e-b06d-4567-b9f8-462fe1e17fa1)

I want the values that are within the exr file since these were the values I process within my program

**Comment by dlemstra** [comment, marked as answer]
The values from that `exr` files are scaled to our quantum range. So that means that `0.0 - 1.0` will be scaled to `0 - 65535` (for a Q16 release)
